adb_shell/adb_device.py

```diff
@@ -74,6 +74,7 @@
 from .adb_message import AdbMessage, checksum, unpack
 from .handle.base_handle import BaseHandle
 from .handle.tcp_handle import TcpHandle
+from .handle.usb_handle import UsbHandle
```

Comment:
This should probably be in a `try: ... except ImportError:` block, in case the user didn’t install the `usb` extras.

Reply:
Yeah, I still don't know the best way to handle `extras` in a python package.
If there's no other way, I'll use what you suggested.

Reply:
I think one solution is

```python
try:
    from .handle.usb_handle import UsbHandle
except ImportError:
    UsbHandle = None
```

And then in `AdbDeviceUsb` you can do:

```python
if UsbHandle is None:
    raise exceptions.InvalidHandleError("To enable USB support you must install this package via `pip install adb-shell[usb]`")
```

Reply:
Changed in https://github.com/Halastra/adb_shell/pull/5